mnemonic/mnemonic.py

```diff
@@ -139,15 +140,13 @@ def to_mnemonic(self, data):
             idx = int(b[i * 11:(i + 1) * 11], 2)
             result.append(self.wordlist[idx])
         if self.detect_language(' '.join(result)) == 'japanese':  # Japanese must be joined by ideographic space.
-            result_phrase = u'\xe3\x80\x80'.join(result)
```

Comment:
This is flat out incorrect.

It is either: `u'\u3000'`(unicode representation) or `'\xe3\x80\x80'` (bytestring representation that needs `.decode('utf8')`)